tests/integration/tests/helpers.py

```diff
@@ -44,3 +47,40 @@ def make_configuration_artifact(
         subprocess.check_call(cmd, shell=True)
     finally:
         os.unlink(configuration_file.name)
+
+
+def make_configuration_apply_script(setup_tester_ssh_connection, script):
+    # Enable writes
+    run(
+        setup_tester_ssh_connection, "mount -o remount,rw /", warn=True,
+    )
+
+    # Install the apply-device-config script
+    tf = NamedTemporaryFile(delete=False)
+    tf.write(script.encode("utf-8"))
+    tf.close()
+
+    try:
+        run(
+            setup_tester_ssh_connection,
+            "mkdir -p /usr/lib/mender-configure",
+            warn=True,
+        )
+        put(
+            setup_tester_ssh_connection,
+            tf.name,
+            key_filename=tf.name,
+            remote_path="/usr/lib/mender-configure/apply-device-config",
+        )
+        run(
+            setup_tester_ssh_connection,
+            "chmod 755 /usr/lib/mender-configure/apply-device-config",
+            warn=True,
+        )
+    finally:
+        os.unlink(tf.name)
+
+    # Disable writes
+    run(
+        setup_tester_ssh_connection, "mount -o remount,ro /", warn=True,
```

Comment:
Another question: Do you actually rely on it being read-only anywhere, or is this just further validation that Configure will work there? I'm asking because maybe it makes sense not to use `warn=True` so that we will know if this step fails. If `warn=True` is set then whatever tests come after this one will unknowingly run with a writeable filesystem.

Reply:
Further validation. You are right, let's remove the warn=True.